c2corg_ui/templates/helpers/view.html

```diff
@@ -74,6 +74,15 @@
   ${get_attr(version, 'comment', md=False)  if version['comment'] else '' | n}
 </%def>
 
+<%def name="show_article_title(locale, is_tab_title=False)">\
```

Comment:
Are you sure we need this?
There's not ``title_prefix`` for articles. You should be able to use the standard ``show_title`` function.